.github/workflows/deploy-app-to-env.yml

```diff
@@ -82,11 +82,21 @@ jobs:
           name: lambda-layers-prisma-client-migration
           path: ./services/app-api/lambda-layers-prisma-client-migration
 
-      - name: Unzip prisma layer
+      - name: Unzip prisma migration layer
         run: |
           tar -C ./services/app-api/lambda-layers-prisma-client-migration -xf ./services/app-api/lambda-layers-prisma-client-migration/nodejs.tar.gz
           rm -rf ./services/app-api/lambda-layers-prisma-client-migration/nodejs.tar.gz
 
+      - uses: actions/download-artifact@v3
```

Comment:
why is this change in the react-scripts PR?

Reply:
Looks like this came in with adf1882. @mojotalantikite @macrael  does that commit look alright to yall just sanity check because we had a bit of funky stuff with app-web merge want to be sure.

Reply:
So yeah, with the webpack5 changes there was a change in behavior that stopped prisma from working again. Basically it was copying in parts of the prisma client, but not any of the engine things that it needs.

In order to get it working again I added a single layer that has the prisma packages and forced webpack from packing up prisma. That way the resolutions for prisma go to the lambda layer, which have the appropriate query engine builds.